### DISCUSSION TITLE
How long do I have to be on the waitlist ?

### DISCUSSION THREAD

**Discussion by Of-qinhai** [community/community#7181] [Copilot Conversations] [answered]
Three weeks after applying for waitlist, I'm still waiting for notice，

**Comment by HerbeMalveillante** [comment]
I applied on june, and I got accepted today. Patience is key !

**Reply by Of-qinhai** [reply]
my godsh ,thank you ：）

**Comment by longer96** [comment]
I also want to have

**Reply by Of-qinhai** [reply]
Up to now, I still haven't obtained the use credentials

**Comment by skholm** [comment]
I was on the waiting list so long that I forgot it even existed. It will take quite a long time.

**Reply by NoamikoT** [reply]
Did you end up getting it? If so, how long did it take?

**Comment by davecheney** [comment, marked as answer]
I'm sorry that you have not been given access yet. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
